packages/mobile/ios/AudiusReactNative/AppDelegate.mm

```diff
@@ -263,4 +263,20 @@ - (Class)getModuleClassFromName:(const char *)name
 
 #endif
 
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
+{
+    if ([[TikTokOpenSDKApplicationDelegate sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation]) {
+        return YES;
+    }
+    return NO;
+}
+
+- (BOOL)application:(UIApplication *)application handleOpenURL:(NSURL *)url
```

Comment:
I think you can delete this too but not sure. I didn't include it for Share Video to TikTok and everything seems to work